types/setting.go

```diff
@@ -354,6 +359,9 @@ var (
 		Required:    true,
 		ReadOnly:    false,
 		Default:     "300",
+		ValueRange: map[string]int{
```

Comment:
`ValueIntRange`, as this is for `SettingTypeInt`

Reply:
Fixed, thanks